cmd/pack3d/main.go

```diff
@@ -28,14 +30,20 @@ func timed(name string) func() {
 }
 
 func main() {
+	var stlsize []fauxgl.Vector
+	//var boundsize fauxgl.Vector
 	var done func()
 
+	boundsize := fauxgl.V(50.0, 50.0, 50.0)
```

Comment:
maybe a  clearer variable name would be 'max_packing_size' or 'maximum_packing_area'  ? There are a lot of bounding boxes :)